MOBN-1170 | Ot custom surgical appointment attributes

shivarach opened the pull request.

kirity-mahanty commented on openmrs/migrations/liquibase.xml:
```diff
@@ -509,6 +509,32 @@
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select count(*) from surgical_appointment_attribute_type where name in ('Surgeon Name', 'Other Surgeon Name',
+                'Anaesthetist technician', 'HIV Status')
```
Keeping similarity to the other fields, 'Anaesthetist technician' should be in Title case like 'Anaesthetist Technician' unless if there is strict requirement from MSF team. If we make the changes, need to do the same in all the relevant places.

shivarach replied:
I've kept them as mentioned in JIRA. I'll check with Swedhan and KC

kirity-mahanty commented on openmrs/migrations/liquibase.xml:
```diff
@@ -509,6 +509,32 @@
+                VALUES('Anaesthetist technician', 'technician', 'java.lang.String' ,'13', 8, NOW(), UUID());
+                INSERT INTO surgical_appointment_attribute_type (name, description, format, sort_weight, creator,
+                date_created, uuid)
+                VALUES('HIV Status', 'HIV Status', 'java.lang.String' ,'14', 8, NOW(), UUID());
```
We can insert multiple rows into a single INSERT statement to avoid some duplicate lines here.

shivarach replied:
OK, will make changes.

shivarach commented:
@kirity-mahanty Find the latest changes.

- Placeholders removed to make it consistent

- Renamed attribute to **Anaesthetist Technician**

- Updated the query to use single INSERT statement.

shivarach commented:
Conflicts resolved.
